net/strongswan/Makefile

```diff
@@ -526,6 +526,7 @@ endef
 define Package/strongswan-swanctl/install
 	$(INSTALL_DIR) $(1)/etc/swanctl/{bliss,ecdsa,pkcs{12,8},private,pubkey,rsa}
 	$(INSTALL_DIR) $(1)/etc/swanctl/x509{,aa,ac,ca,crl,ocsp}
+	$(INSTALL_DIR) $(1)/etc/swanctl/conf.d
```

Comment:
Move 1 line up so that this group of INSTALL_DIR statements maintains alphabetical order. Alternatively include it in the first INSTALL_DIR line.

Reply:
Done.